### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by rodrigopenayo98**
This Pull Request includes the necessary changes to the "Bookstore-rod" project to meet the following requirements:

> 1. Added missing JSX elements to complete the website layout, without introducing additional functionality.
> 2. Adjusted the code to precisely match the design specifications provided in Zeplin.
> 3. Applied the detailed CSS rules from Zeplin to ensure the website has the same style and appearance as defined in the design.

All changes have been thoroughly tested and reviewed to ensure the project functions correctly and the design aligns with Zeplin's guidelines.

The PR is ready for review and merging into the main repository. Any further feedback or suggestions will be taken into consideration, and we are available to make any necessary adjustments. Thank you for your attention!

### CODE REVIEW

File: src/components/BookForm.js [hunk 1]
```diff
@@ -28,22 +28,31 @@ const BookForm = () => {
...
-        <Button text="Add Book" handleAddBook={handleAddBook} />
+        <div className="input-container">
+          <div className="input-author-book-container">
+            <input
+              className="input-author-book"
+              type="text"
+              placeholder="Category"
+              value={author}
+              onChange={(e) => setAuthor(e.target.value)}
+            />
```
Comment (andersonlebon) [inline src/components/BookForm.js]: - [ ] Nice work on the style I can see it's wonderful, how ever I think the category of the book should be taken form a drompdown list or `<select><select>` tag rather that a simple input

File: src/App.js [hunk 2]
```diff
@@ -1,4 +1,3 @@
-import React from 'react';
 import { BrowserRouter, Route, Routes } from 'react-router-dom';
 import { Provider } from 'react-redux';
 import store from './redux/store';
```
Comment (andersonlebon) [inline src/App.js:3]: ## [OPTIONAL]

- [ ] Kindly make sure the your app is using the API correctly, you should not create an new API every time the app is refreshed, kindly it need to be created once 

```jsx
// the below code should only have bein only used once, the data in the API should not not desapear when the page is refreshed,
import axios from 'axios';

const id = 'TUBThKZKOHdsLlkMqwTD';

const api = axios.create({
  baseURL: `https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/${id}`,
});

export default api;
```

- [ ] KIndly make sure also the 3 books that was use to intiate the state are removed, they are not neeeded as we have already an API to store our books.

- [ ] As the API is having a field to hold the auther of the book, I would sugget adding it also, as is importent for the Library recored

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @rodrigopenayo98 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 
- NIce design ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookForm.js (hunk 1 above)
Comment (rodrigopenayo98) [reply src/components/BookForm.js]: @andersonlebon Hi! the dropdown functionality is not a requirement, as indicated in the project guides, just recreate the arrow inside the input as in the layout, if you read the requirements it says to add new elements to the project but without its functionality: 

> 1. Add any missing JSX elements to your project (without adding extra functionality)
> 2. Your final design should match Zeplin's design (this includes all elements + the same styling)
> 3. You may use the CSS rules detailed in Zeplin to style the website.

the dropdown should not work as such, it should just be there.

Comment (andersonlebon) [reply src/components/BookForm.js]: Hey @rodrigopenayo98, eventhough the requirement does not show directly that the dropdown/select input should be there, the design guidelines show it(`the design should match the Zeplin's design including all elements ....`), therefore at a place of a select input we should not use a text input.

**Review by frank1738 — APPROVED** [review]
Hi @rodrigopenayo98 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ App is designed as expected 👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**